### DISCUSSION TITLE
Product feed stuck IN_PROGRESS

### DISCUSSION THREAD

**Discussion by silvialort** [amzn/selling-partner-api-models#3862] [General]
We've been using SP-API to upload feeds for almost a year now. We hadn't have any problems until 3 weeks ago, when suddenly the feeds got stucked IN_PROGRESS and never got DONE. We contacted with the Amazon Seller Support and we got to cancel most of the feeds.
However, one of the feeds, specifically a Bookloader feed, became stuck in the IN_PROGRESS status, and we have been unable to cancel it via the API because it returns a **400 error with the code -37**. We've reached out to support team on multiple occasions, but they told us that the document feed doesn't exist and therefore, they can't cancel it either. With that feed stuck IN_PROGRESS, it's impossible to upload new feeds, as they appear with IN_QUEUE status.
Has someone encountered a similar problem with the API?
Thanks in advance!

**Comment by xiaolei8587** [comment]
Hello, could you please clarify if this issue has been resolved? If so, may I ask by what means it was resolved? Thank you.

**Comment by ritikacd** [comment]
Hi @silvialort 

Please submit a ticket with the SP-API Developer support team using the guidelines [here](https://developer.amazonservices.com/support). They should be able to investigate into the stuck feed. 

Thanks,
Ritika
Selling Partner API Services
